docs/gadgets/trace/capabilities.md

```diff
@@ -142,6 +142,117 @@ You can now delete the pod you created:
 $ kubectl delete -f docs/examples/app-set-priority-locked-down.yaml
 ```
 
+#### Interpreting advanced columns
+
+Some columns are not displayed by default:
+* `caps`: the effective capability bitfield from the objective context of the process
+* `capsnames`: same as caps in a human friendly format
+* `currentuserns`: the user namespace of the process
+* `targetuserns`: the user namespace that the kernel used to test the
+  capability.
+
+They can be useful to understand advanced usage of capabilities.
+Let's see two examples.
+
+```
+$ kubectl run -ti --rm --restart=Never \
+    --image busybox --privileged testcaps -- \
+    chroot /
+```
+
+```
+$ kubectl gadget trace capabilities -o json | jq .
+{
+  "node": "minikube-docker",
+  "namespace": "default",
+  "pod": "testcaps",
+  "container": "testcaps",
+  "timestamp": 1676910664324109724,
+  "type": "normal",
+  "pid": 1202036,
+  "comm": "chroot",
+  "syscall": "chroot",
+  "cap": 18,
+  "capName": "SYS_CHROOT",
+  "audit": 1,
+  "verdict": "Allow",
+  "insetid": false,
+  "mountnsid": 4026533229,
+  "targetuserns": 4026531837,
+  "currentuserns": 4026531837,
+  "caps": "3fffffffff",
```

Comment:
I think it needs to be updated after last change, now it's printed as an integer. Perhaps it's better to use `-o columns` to avoid user dealing with json.

Reply:
With -o custom-columns I cannot get the full field because it gets ellipsed. I don't have a monitor big enough to enlarge the terminal. And `export COLUMNS=400` does not seem to work.
cc @flyth 

I will update the output with json.